common-rendering/src/components/FirstPublished.tsx

```diff
@@ -8,9 +8,11 @@ const padString = (time: number) => (time < 10 ? `0${time}` : time);
 const FirstPublished = ({
 	firstPublished,
 	blockLink,
+	showTimeAgo = true,
 }: {
 	firstPublished: number;
 	blockLink: string;
+	showTimeAgo?: boolean;
```

Comment:
What about changing this to `isPinnedPost`? That would feel more declarative and less imperative (I love using those words in PR reviews). Essentially, it's less tell me what to do and more tell me what's happening and let me decide how to react

I'm a little 🤷  here though and you know this code better than me so if you have strong opinions then I'm happy to defer to you